src/dsp_tools/cli/create_parsers.py

```diff
@@ -121,7 +121,7 @@ def _add_start_stack(subparsers: _SubParsersAction[ArgumentParser]) -> None:
     subparser.add_argument(
         "--latest",
         action="store_true",
-        help="use the latest dev version of DSP-API, from the main branch of the GitHub repository",
+        help="use the latest dev version of DSP-API/DSP-APP, from the main branch of the GitHub repository",
```

Comment:
This is a user-friendly abstraction of what is really happening. Already on line 110, we don't tell the entire truth.